.travis.yml

```diff
@@ -57,7 +57,7 @@ install:
   - "inveniomanage config create secret-key"
   - "inveniomanage config set CFG_EMAIL_BACKEND flask_email.backends.console.Mail"
   - "inveniomanage config set CFG_BIBSCHED_PROCESS_USER `whoami`"
-  - "inveniomanage config set PACKAGES ['invenio.base', 'invenio.modules.accounts', 'invenio.modules.records', 'invenio_tags']"
+  - "inveniomanage config set PACKAGES ['invenio_base', 'invenio.modules.accounts', 'invenio.modules.records', 'invenio_tags']"
```

Comment:
can you please update `.travis.yml` based on `invenio-accounts` and and `.travis.invenio.cfg` with current configuration?

Please note that we need to update 'invenio.modules.accounts' and 'invenio.modules.records' too.